README.md

```diff
@@ -112,11 +112,11 @@ User.where('id', 1).fetch({withRelated: ['posts.tags']}).then(function(user) {
 * [Virtuals](https://github.com/bookshelf/bookshelf/wiki/Plugin:-Virtuals): Define virtual properties on your model to compute new values.
 * [Visibility](https://github.com/bookshelf/bookshelf/wiki/Plugin:-Visibility): Specify a whitelist/blacklist of model attributes when serialized toJSON.
 * [Pagination](https://github.com/bookshelf/bookshelf/wiki/Plugin:-Pagination): Adds `fetchPage` methods to use for pagination in place of `fetch` and `fetchAll`.
-* [Case Converter](https://github.com/bookshelf/bookshelf/wiki/Plugin:-Case-Converter): Handles the conversion between the database's snake_cased and a model's camelCased properties automatically.
```

Comment:
This particular Case Converter plugin *is* part of core, so it can't be removed from this list: https://github.com/bookshelf/bookshelf/blob/master/lib/plugins/case-converter.js. It's just missing a dedicated wiki page.

Reply:
I understand, just missing the page in the wiki.